app/views/recipes/public.html.erb

```diff
@@ -0,0 +1,23 @@
+<h2 class="text-center">Public Recipes</h2>
+
+<div id="recipes">
+  <% @recipes.each do |recipe| %>
+      <div class="container-fluid">
+        <div class="d-flex justify-content-around align-items-center border m-2">
+          <div class="m-3">
+            <p class="fw-bold"><%= recipe.name %></p>
+            <p><%= recipe.user.name %></p>  
+          </div>
+          <div class="m-3">
+            <% if recipe.public == false%>
+                <p><%= recipe.decription %></p>
+            <% else %>
+                <p>Total foods items: <%= RecipeFood.where(recipe_id: recipe.id).count %></p>
+                <% price = RecipeFood.where(recipe_id: recipe.id).reduce(0) { |sum, f| sum + (f.quantity*Food.find(f.food_id).price).to_f } %>
+                <p>Total price: $<%= price %></p>
+            <% end %>
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=in%20the%20wireframe.-,Should%20lead%20to%20recipe%20details.,-Recipe%20details%3A) when you click on a recipe item should lead to recipe details. So please fix that.

Reply:
@Omar-Muhamad, Thanks for your review. We have fixed it.